### PR TITLE
[editor] Minor refactoring: do not copy FeatureType to save FeatureID + style fixes.

### PR DESCRIPTION
**Pull request by tatiana-yan**

### CODE REVIEW

File: editor/edits_migration.hpp [hunk 1]
```diff
@@ -8,18 +8,18 @@
...
+#include <functional>
 
 namespace editor
 {
 DECLARE_EXCEPTION(MigrationError, RootException);
 
-using TGenerateIDFn = function<FeatureID()>;
+using GenerateIDFn = std::function<FeatureID()>;
 
 /// Tries to match xml feature with one on a new mwm and retruns FeatrueID
```
Comment (mpimenov) [inline editor/edits_migration.hpp]: s/retruns/returns/

Comment (mpimenov) [inline editor/edits_migration.hpp]: s/Featrue/Feature/

File: editor/edits_migration.hpp [hunk 2]
```diff
@@ -8,18 +8,18 @@
...
 /// of a found feature, thows MigrationError if migration fails.
```
Comment (mpimenov) [inline editor/edits_migration.hpp]: s/thows/throws/

File: editor/edits_migration.cpp [hunk 3]
```diff
@@ -87,26 +86,26 @@ FeatureID MigrateWayOrRelatonFeatureIndex(
...
       someFeaturePoint);
 
   if (count == 0)
     MYTHROW(MigrationError, ("No ways returned for point", someFeaturePoint));
 
-  if (!feature)
+  if (!fid)
   {
     MYTHROW(MigrationError,
             ("None of returned ways suffice. Possibly, the feature have been deleted."));
```
Comment (mpimenov) [inline editor/edits_migration.cpp]: s/have/has/

File: editor/edits_migration.hpp (hunk 1 above)
Comment (tatiana-yan) [reply editor/edits_migration.hpp]: done

Comment (tatiana-yan) [reply editor/edits_migration.hpp]: done

File: editor/edits_migration.hpp (hunk 2 above)
Comment (tatiana-yan) [reply editor/edits_migration.hpp]: done

File: editor/edits_migration.cpp (hunk 3 above)
Comment (tatiana-yan) [reply editor/edits_migration.cpp]: done

**Comment by tatiana-yan** [comment]
@mpimenov PTAL

**Pull request merged**